packages/ocamlformat-bench/ocamlformat-bench.0.20.0/opam

```diff
@@ -0,0 +1,57 @@
+opam-version: "2.0"
+synopsis: "Auto-formatter for OCaml code"
+description:
+  "OCamlFormat is a tool to automatically format OCaml code in a uniform style."
```

Comment:
The description is a bit surprising. Is this an internal package?

Reply:
Also if this is meant to be a public package, the license is missing

Reply:
Yes it's an internal package, I will remove this file. It was somehow necessary to add an opam file when we started to use `current-bench` but I forgot to update our release workflow.

Reply:
Does `current-bench` not work with just having `(package (name ocamlformat-bench) ...)` without autogenerating its opam file?

Reply:
I will have a look